src/Api/DevelopmentEndpoints.cs

```diff
@@ -38,5 +39,8 @@ public static void UseDevelopmentEndpoints(this WebApplication app)
 
             return "All OK. Happy Developing!";
         });
+        
+        app.MapGet("health/throw/api", (ctx) => throw new ApiException("test_error", "Testing error response", 400));
+        app.MapGet("health/throw/exception", (ctx) => throw new Exception("Testing error response", new Exception("Inner exception")));
     }
```

Comment:
I think it would be beneficial if we had these endpoints mapped even in prod -- they're useful to verify both error tracking in prod as well as SDK behaviours

Reply:
Ok moved them back.